scanner/internal/httputil/transport_deny.go

```diff
@@ -0,0 +1,25 @@
+package httputil
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/quay/zlog"
+	"github.com/stackrox/rox/pkg/httputil"
+)
+
+var (
+	errTrafficDenied = errors.New("HTTP traffic denied")
+
+	// DenyTransport returns a http.RoundTripper which denies all requests.
+	DenyTransport http.RoundTripper = httputil.RoundTripperFunc(denyTransport)
+)
+
+func denyTransport(req *http.Request) (*http.Response, error) {
+	zlog.Error(context.Background()).
+		Str("method", req.Method).
+		Str("url", req.URL.String()).
+		Msg("denied HTTP request")
+	return nil, errTrafficDenied
+}
```

Comment:
Should we consider having this panic in non-release builds (ie: `utils.Should`) so that we catch when libVuln is reaching out?  Or perhaps add some check in CI that looks for this particular log message and abends accordingly?